### ISSUE TITLE
RCTConvert+MapKit.h

### ISSUE THREAD

**Issue by karpoff** [airbnb/react-native-maps#1193]
https://github.com/airbnb/react-native-maps/blob/master/lib/ios/AirMaps.xcodeproj/project.pbxproj
it still have links to RCTConvert+MapKit.h/m (whick was renamed to RCTConvert+AirMap.h/h here https://github.com/airbnb/react-native-maps/commit/4926f5b53fac8e1ab9aaf877e81048e262dee8aa)
it causes ios build errors

changing AirMaps project file (removig old files and adding new ones) can be used as workaround

**Comment by doananh234** [comment]
open libraries -> right click AirMaps.xcodeproj -> add file to AirMaps -> go to find react-native-maps into node_module -> lib -> ios -> AirMaps -> select RCTConvert+MapKit.h/m -> add

maybe help you.

**Comment by karpoff** [comment]
@doananh234 those files were replaced by RCTConvert+AirMap.h/m 
https://github.com/airbnb/react-native-maps/tree/master/lib/ios/AirMaps

**Comment by Panda313** [comment]
I got the same error as you. I Just did what @doananh234 suggested but instead of adding "RCTConvert+MapKit.h/m" I looked for "RCTConvert+AirMap.h/m" inside node_modules/react-air-maps.

The files were actually in red inside xcode, certainly not well linked.

Anyway, did the trick for me

**Comment by karpoff** [comment]
yep. this way works for me to
open AirMaps project inside Libraries folder
remove reference of RCTConvert+MapKit.h/m
add reference to RCTConvert+AirMap.h/m (from node_modules/react-native-maps/lib/ios/AirMaps)

**Comment by mhaagens** [comment]
@karpoff  has the correct solution. Go to Libraries/AirMaps.xcodeproj/AirMaps/ and remove the references to RCTConvert+MapKit.h/m and then add RCTConvert+AirMap.h/m.

**Comment by agentilela** [comment]
Wrote a simple shell script to fix react-native-maps build errors that gets run whenever i rebuild my project. Maybe it'll be helpful to someone:

`sed -ie 's/RCTConvert\+MapKit\.m/RCTConvert\+AirMap\.m/g' node_modules/react-native-maps/lib/ios/AirMaps.xcodeproj/project.pbxproj`
`sed -ie 's/RCTConvert\+MapKit\.h/RCTConvert\+AirMap\.h/g' node_modules/react-native-maps/lib/ios/AirMaps.xcodeproj/project.pbxproj`
`rm -f node_modules/react-native-maps/.babelrc`

**Comment by JonatanSalas** [comment]
@agentilela your solution works like a charm!

**Comment by creeperyang** [comment]
@karpoff thanks for your solution. And my env is

```
"dependencies": {
    "react": "16.0.0-alpha.6",
    "react-native": "0.43.4",
    "react-native-maps": "^0.14.0"
  }
```

Just remove `RCTConvert+MapKit.h/m` and add `RCTConvert+AirMap.h/m`.

<img width="234" alt="2017-04-21 6 14 29" src="https://cloud.githubusercontent.com/assets/8046480/25273444/6877a64c-26be-11e7-95d5-093f8e867d55.png">

**Comment by prontiol** [comment]
This is not correct solution, because you have to modify Xcode Project from the `node_modules` folder to make it work. So you basically end up with non-working code in your repo and any other team member will fail to run the project from scratch. Also CI will fail.

**Comment by agentilela** [comment]
I added it to my postinstall scripts so it doesn't break CI. But yeah, it's
not the correct solution. We need a version with the fixes merged.

**Comment by npomfret** [comment]
Is there a fix on the way for this?

**Comment by prontiol** [comment]
@npomfret There is a PR #1195.
Also, I have forked and merged this PR in my repo [master](https://github.com/prontiol/react-native-maps/releases/tag/hotfix), so you can use it for now, like this:
```
yarn add prontiol/react-native-maps#hotfix
```
Don't use this in production, though.

**Comment by itinance** [comment]
Anything we can do? We faced this issue also on a green field application based on RN 0.45

**Comment by christopherdro** [comment]
This was closed because the fix has been merged via https://github.com/airbnb/react-native-maps/commit/4b861c949a079a393f48a485943d3a506a151ac0 released in version 0.15.0.

@itinance If your application is currently on RN 0.45 then you'll have to update to 0.15+
